gulp-tasks/utils/rollup-helper.js

```diff
@@ -18,7 +19,7 @@ module.exports = {
   // plugin set up and used to ensure as consist set of tests
   // as possible.
   getDefaultPlugins: (buildType, {module = false} = {}) => {
-    const plugins = [];
+    const plugins = [resolve()];
```

Comment:
Is this an intentional change? If so, what's it used for?

Reply:
Yeah, this is because I'm loading `logger.mjs` and `Deferred.mjs` from `workbox-core` now, and since that's a `node_module` dependency, I need to use `rollup-plugin-node-resolve` or Rollup will error.

The reason this isn't needed for other Workbox packages is all SW packages [treat other package dependencies as external](https://github.com/GoogleChrome/workbox/blob/07bef20/gulp-tasks/utils/build-browser-bundle.js#L29-L90), thus there's nothing to resolve.